docs/authentication-flow.md

```diff
@@ -0,0 +1,46 @@
+# Authentication Flow
+
+This document will detail the authentication flow Surfbird is following at the time of writing (and will be updated accordingly with changes)
+
+### Account Request
+
+As soon as the renderer process is loaded and the Vue mount is initialized, the process will send out a request to the main process using the
+channel `surfbird:request:accounts`.
+
+The main process is then returning all already authenticated accounts in an array over the `surfbird:get:accounts` channel. Now here are two
+possible outcomes from this request:
+
+1. If there are accounts available, they will be added to the store so that they are accessible in the renderer process
+2. If no accounts were returned by the request, a modal prompting the user to authenticate with a service will be opened
+
+### Authentication
+
+If no account has been authenticated to Surfbird prior, a modal prompting the user to authenticate with a service will be opened.
```

Comment:
This is repetition of the previous line (L14), although looks intentional... and makes sense if you link someone directly to _Authentication_ — is it intentional?

Reply:
This is intentional, the document is supposed to be read from top-to-bottom, so we follow with the case of no account being available, thus someone having to authenticate.

The section below (Account Addition) mentions that both outcomes from above (in relation to the used IPC channel name) will lead there.